Wangscape/Options.h

```diff
@@ -17,13 +20,20 @@ class Options
     typedef std::vector<Clique> CliqueList;
 
     std::string filename;
+    std::string directory;
```

Comment:
If so, maybe the best idea is to define it inside Options, as public class, `Options::Location` or similar.